setup.cfg

```diff
@@ -60,3 +61,7 @@ license_files =
 python_requires = >= 3.6
 packages = find:
 zip_safe: False
+
+[options.extras_require]
+stripe =
```

Comment:
We should make this optional, lots of other projects don't use stripe

So we can install `django-ckc` without also installing stripe, unless we do `django-ckc[stripe]` or something